app/views/files/index.slim

```diff
@@ -19,6 +19,8 @@ ul
 					input type="hidden" name="_method" value="DELETE"
 					button type="submit" Delete
 
+				a href="#{file.route}?download=true" Download
+
```

Comment:
I definitely think we should break apart this block here... just not sure the best way to do it.